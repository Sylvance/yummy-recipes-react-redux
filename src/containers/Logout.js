import React from 'react'
import PropTypes from 'prop-types'
import { bindActionCreators } from 'redux'
import { connect } from 'react-redux'

import LogoutComponent from '../components/LogoutComponent'

import * as Actions from '../actions'

const Logout = ({actions}) => (
  <div>
    <LogoutComponent logoutUser={actions.logoutUser} />
  </div>
)

Logout.propTypes = {
  actions: PropTypes.object.isRequired
}

const mapStateToProps = state => ({
  authentication: state.auth
})

const mapDispatchToProps = dispatch => ({
  actions: bindActionCreators(Actions, dispatch)
})

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(Logout)
