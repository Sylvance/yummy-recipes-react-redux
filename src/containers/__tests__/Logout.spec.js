// Auto-generated do not edit

/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Logout from '../Logout';


describe('Logout test', () => {
  it('Logout should match snapshot', () => {
    const component = renderer.create(
        <div>
      </div>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
