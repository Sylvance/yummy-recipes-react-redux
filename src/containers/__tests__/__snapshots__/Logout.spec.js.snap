// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout test Logout should match snapshot 1`] = `<div />`;
